### PR TITLE
French translation proposal

### PR DESCRIPTION
**Pull request by seigneurfuo**
Hello.
Thank for creating this application. I really like it. That's why I've decided to translate it into French, so that other people can use it too.

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <string name="error_oreo_shortcut_host_permission">Fast Draw n\'a pas l\'autorisation d\'héberger des raccourcis. Assurez-vous que Fast Draw est définit comme lanceur par défaut.</string>
+    <string name="error_oreo_shortcut_host_permission">Fast Draw n\'a pas l\'autorisation d\'héberger des raccourcis. Assurez-vous que Fast Draw est le lanceur par défaut.</string>
+    <string name="error_oreo_shortcut_host_permission">Fast Draw n\'a pas l\'autorisation d\'héberger des raccourcis. Assurez-vous que Fast Draw est définit comme lanceur par défaut.</string>
+    <string name="error_expand_notifications_panel">Impossible d\'ouvrir le panneau des notifications. Votre version d\'Android ne supporte peut-être pas cela avec les lanceurs tiers</string>
+    <string name="error_launch_exception">Impossible lancer l\'élément à cause d\'une erreur inconnue. Veuillez signaler ce problème.</string>
+    <string name="error_expand_notifications_panel">Impossible d\'ouvrir le panneau des notifications. Votre version d\'Android ne supporte peut-être pas cela avec les lanceurs tiers</string>
```
Comment (peterfajdiga) [inline app/src/main/res/values-fr/strings.xml:51]: There are multiple entries for `error_oreo_shortcut_host_permission` and `error_expand_notifications_panel`.

**Review by peterfajdiga — CHANGES_REQUESTED** [review]
Hi, thanks for the translation! There is one small problem, though, see the other comment.

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (seigneurfuo) [reply app/src/main/res/values-fr/strings.xml]: Je pense avoir eu un problème avec Android Studio. Je n'ai pas fait attention lors du commit. Le problème est corrigé. Merci.

**Review by peterfajdiga — APPROVED** [review]
Merci!

**Pull request merged**